server/spec/expectations/actions.js

```diff
@@ -0,0 +1,50 @@
+const Action = require('../../models/action');
+const { checkRecord, expect, toArray } = require('../utils');
+
+module.exports = function(actual, expected) {
+
+  expect(actual, 'res.body').to.be.an('object');
+
+  const expectedKeys = [ 'id', 'title', 'description', 'themeId', 'createdAt', 'updatedAt' ];
+
+  expect(actual, 'res.body').to.have.all.keys(expectedKeys);
+
+  if (expected.id) {
+    expect(actual.id, 'action.id').to.equal(expected.id);
+  } else {
+    expect(actual.id, 'action.id').to.be.a('string');
+  }
+
+  expect(actual.title, 'action.title').to.equal(expected.title);
+
+  expect(actual.description, 'action.description').to.equal(expected.description);
+
+  expect(actual.themeId, 'action.themeId').to.be.a('string');
```

Comment:
The exact value should be checked here, not just the type. `getExpectedAction` can automatically add it from the related model.